src/utils/currencies.ts

```diff
@@ -53,12 +53,13 @@ function getFixedTokenAmount(amount: string, decimals = 18): string {
 function getAmountInPreferredCurrency(
   usdAmount: number,
   preferredCurrency: string,
-  fiatExchangeRates: FiatExchangeRates
-): number {
+  fiatExchangeRates?: FiatExchangeRates
```

Comment:
Since there's a case where the `FiatExchangeRates` are not used (when `preferredCurrency === CurrencySymbol.USD.code`) we agreed on leaving `fiatExchangeRates` param as optional.